SQL/0000-00-03-ConfigTables.sql

```diff
@@ -44,30 +44,31 @@ CREATE TABLE `Config` (
 --
 
 INSERT INTO ConfigSettings (Name, Description, Visible, AllowMultiple, Label, OrderNumber) VALUES ('study', 'Settings related to details of the study', 1, 0, 'Study', 1);
-INSERT INTO ConfigSettings (Name, Description, Visible, AllowMultiple, DataType, Parent, Label, OrderNumber) SELECT 'additional_user_info', 'Display additional user profile fields on the User accounts page (e.g. Institution, Position, Country, Address)', 1, 0, 'boolean', ID, 'Additional user information', 15 FROM ConfigSettings WHERE Name="study";
```

Comment:
There seems to be a lot of regressions in this patch. (At least issue_tracker_url->mantis_url and phantom regexes default configs, but I might have missed others in all the changes in this patch that aren't related to consent..)

Reply:
@ridz1208 please explain

Reply:
@driusan good catch, might need to be done all over again then

Reply:
@zaliqarosli There are things that were merged in other pull requests which are being undone by this change

Reply:
i made changes to this if you guys want to have a look